axelor-project/src/main/resources/views/ProjectTask.xml

```diff
@@ -52,6 +52,59 @@
       onClick="action-project-task-validate-delete-team-task-confirmation, action-project-task-method-delete-project-task"/>
   </grid>
 
+  <grid name="project-task-editable-grid" title="Tasks"
```

Comment:
Can't these 2 views be merged ? With a value in context on action view that show the project or no for exemple.

Reply:
project-task-editable-grid is editable grid 
project-task-tree-project-grid is Tree Grid with tree-grid widget 

I don't think we can merge those 2 behaviours